src/spikeinterface/preprocessing/motion.py

```diff
@@ -418,6 +420,28 @@ def correct_motion(
 correct_motion.__doc__ = correct_motion.__doc__.format(_doc_presets, _shared_job_kwargs_doc)
 
 
+def save_motion_info(motion_info, folder, overwrite=False):
```

Comment:
OK for this but then correct_motion() should use it internally to avoid code duplication.

Reply:
done